arwen/contexts/managedType.go

```diff
@@ -297,15 +297,15 @@ func (context *managedTypesContext) GetTwoBigFloats(handle1 int32, handle2 int32
 }
 
 // PutBigFloat adds the given value to the current values map and returns the handle
-func (context *managedTypesContext) PutBigFloat(value float64) int32 {
+func (context *managedTypesContext) PutBigFloat(value *big.Float) int32 {
```

Comment:
please add a test that calls this function like:
```GO
//....
managedTypesContext.PutBigFloat(nil)
//...
```

Then add the required fixes.